UnitTests/Configuration/JsonConverterTests.cs

```diff
@@ -251,9 +251,9 @@ public void TestColorSchemesSerialization ()
 	public class KeyJsonConverterTests {
 		[Theory, AutoInitShutdown]
 		[InlineData (Key.A, "A")]
-		[InlineData (Key.a | Key.ShiftMask, "a, ShiftMask")]
+		[InlineData (Key.A | Key.ShiftMask, "A, ShiftMask")]
 		[InlineData (Key.A | Key.CtrlMask, "A, CtrlMask")]
-		[InlineData (Key.a | Key.AltMask | Key.CtrlMask, "a, CtrlMask, AltMask")]
```

Comment:
I think you also should use the lower case char to prove that the also can be used on shortcuts.

```cs
	[Theory, AutoInitShutdown]
	[InlineData (Key.A, "A")]
        [InlineData (Key.A | Key.ShiftMask, "A, ShiftMask")]
	[InlineData (Key.A | Key.CtrlMask, "A, CtrlMask")]
	[InlineData (Key.A | Key.AltMask | Key.CtrlMask, "A, CtrlMask, AltMask")]
        [InlineData((Key)'a' | Key.AltMask | Key.CtrlMask, "Space, A, CtrlMask, AltMask")]
        [InlineData((Key)'a' | Key.ShiftMask, "Space, A, ShiftMask")]
        [InlineData (Key.Delete | Key.AltMask | Key.CtrlMask, "Delete, CtrlMask, AltMask")]
	[InlineData (Key.D4, "D4")]
	[InlineData (Key.Esc, "Esc")]
	public void TestKeyRoundTripConversion (Key key, string expectedStringTo)

```